packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraSettingsTests.m

```diff
@@ -0,0 +1,256 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import camera_avfoundation;
+@import camera_avfoundation.Test;
+@import XCTest;
+@import AVFoundation;
+#import <OCMock/OCMock.h>
+#import "CameraTestUtils.h"
+#import "MockFLTThreadSafeFlutterResult.h"
+
+static const char *TEST_RESOLUTION_PRESET = "medium";
+static const int TEST_FPS = 15;
+static const int TEST_VIDEO_BITRATE = 200000;
+static const int TEST_AUDIO_BITRATE = 32000;
+static const bool TEST_ENABLE_AUDIO = YES;
+
+@interface CameraSettingsTests : XCTestCase
+@property(readonly, nonatomic) FLTCam *camera;
+@end
+
+@implementation CameraSettingsTests
+
+/// Expect that optional positive numbers can be parsed
+- (void)testParsingPositiveNumberOrNull {
+  // extern function from CameraPlugin.m
+  NSNumber *toPositiveOrNull(NSNumber * number);
```

Comment:
Things should not be redeclared in test files; if they need to be visible for testing, they should be a in a `..._Test.h` header. See FLTSavePhotoDelegate_Test.h, for example.

Reply:
Declaration is moved to ..._Test.h